Symfony/CS/Fixer/Symfony/ExtraEmptyLinesFixer.php

```diff
@@ -76,4 +117,75 @@ public function getPriority()
         // should be run after the UnusedUseFixer
         return -20;
     }
+
+    private function removeBetweenUse($index)
+    {
+        // remove empty lines between two `use`s
+        $tokenCount = count($this->tokens);
+        for ($i = $index; $i < $tokenCount; ++$i) {
+            if ($this->tokens[$i]->equals(';')) {
+                $next = $this->tokens->getNextMeaningfulToken($i);
+                if ($this->tokens[$next]->isGivenKind(T_USE)) {
+                    for ($i = $next; $i > $index; --$i) {
+                        if ($this->tokens[$i]->isWhitespace() && substr_count($this->tokens[$i]->getContent(), "\n") > 1) {
+                            $this->tokens[$i]->setContent("\n");
+                        }
+                    }
+                }
+
+                break;
```

Comment:
For me it would be more readable with more `break`s and `continue`s..

```php
for ($i = $index; $i < $tokenCount; ++$i) {
    if (!$this->tokens[$i]->equals(';')) {
        continue;
    }
    
    $next = $this->tokens->getNextMeaningfulToken($i);
    if (!$this->tokens[$next]->isGivenKind(T_USE)) {
        break;
    }
    
    for ($i = $next; $i > $index; --$i) {
        if ($this->tokens[$i]->isWhitespace() && substr_count($this->tokens[$i]->getContent(), "\n") > 1) {
            $this->tokens[$i]->setContent("\n");
        }
    }
    break;
}
```